components/atom/image/src/defaults/ImageNotFoundIcon.js

```diff
@@ -0,0 +1,16 @@
+          id='path-1'
+        />
+      </g>
+    </svg>
+  )
+}
+
+export default ImageNotFoundIcon
```

Comment:
In other components, we use to import a SUI-icon as a default icon
https://github.com/SUI-Components/svg-iconset

there you have some examples: 
https://github.com/SUI-Components/sui-components/blob/master/components/breadcrumb/basic/src/index.js#L5
https://github.com/SUI-Components/sui-components/blob/master/components/molecule/notification/src/index.js#L4

To be more consistent, i would use this way.
The problem is that we does not have the icon you need in this library, so let's merge it and we'll discuss about how to proceed in a future similar situations.

right?